<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TowerWindowClass</class>
 <widget class="QDialog" name="TowerWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TowerWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>B点</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineB">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>截面</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>C点</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineC">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSection">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxMaterial">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>材料</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>桥塔类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxType">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>右侧</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>左侧</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>桥塔索点</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textLeftTower">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>45, 1.2, 3@1.5, 3@2, 2@2.3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textRightTower">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>420</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>45, 1.2, 3@1.5, 3@2, 2@2.3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>桥塔高度</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSure">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>480</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonCancel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>480</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineH1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineH2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineB</tabstop>
  <tabstop>lineC</tabstop>
  <tabstop>lineH1</tabstop>
  <tabstop>lineH2</tabstop>
  <tabstop>comboBoxSection</tabstop>
  <tabstop>comboBoxMaterial</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>textLeftTower</tabstop>
  <tabstop>textRightTower</tabstop>
  <tabstop>buttonSure</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>buttonCancel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
